net/tn5250/Portfile

```diff
@@ -1,33 +1,29 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
+github.setup        tn5250 tn5250 0.17.6 v
+github.tarball_from releases
 name                tn5250
-version             0.17.4
 revision            1
 categories          net
 license             LGPL-2.1+
 platforms           darwin macosx freebsd
-maintainers         nomaintainer
-
+maintainers         snuxoll openmaintainer
 description         tn5250 terminal emulator
 
 long_description    tn5250 is a telnet client for the IBM iSeries and AS/400 \
                     that emulates 5250 terminals and printers.
 
-homepage            http://tn5250.sourceforge.net/
-
-master_sites        sourceforge:project/tn5250/tn5250/${version}
-
-checksums           sha1    2c84f03f798fd5095009d6798d1e6c0b29e48a75 \
-                    rmd160  a2fd1d96fe89a521c53449174b0898574562ad7b
+homepage            http://tn5250.github.io/
 
-patchfiles          configure.patch
+checksums           rmd160  5b788fe4bd46ec446d30924bdbb6c89cace73e88 \
+                    sha256  f6383bc62360d858d55337296f373fb1797a2ba39dc931cd09d64a4a558e1641 \
+                    size    700576
 
-# from https://svnweb.freebsd.org/ports/head/net/tn5250/
-patchfiles-append   patch-lib5250_sslstream.c
 
-configure.args      --enable-ssl
+configure.args      --with-ssl
 
 #variant python {
```

Comment:
remove the commented out part here or re-add the variant if this is of interest (but then of course with a modern version).